dev/integration_tests/channels/ios/Flutter/Debug.xcconfig

```diff
@@ -0,0 +1,4 @@
+#include "Generated.xcconfig"
+#include "Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig"
```

Comment:
For instance, this file isn't checked in. So this project isn't buildable until the user runs pod install once though I suspect this project doesn't conceptually depend on cocoapods or the pub plugin system